app/templates/components/comment-component.hbs

```diff
@@ -1,5 +1,5 @@
 <div>
-  <p>{{comment.content}}</p>
+  <p>{{{comment.content}}}</p>
```

Comment:
This is fine for the workshop but it's probably worth mentioning during it that this is essentially opening you up to XSS 🎃

Reply:
Absolutely, but I feel it's not worth the effort of doing proper sanitisation here. We can stick a big red warning on the slide for this bit.

Reply:
Yep, definitely agree 👍